src/main/java/com/uid2/admin/vertx/service/ClientKeyService.java

```diff
@@ -508,6 +516,11 @@ private void handleClientRename(RoutingContext rc) {
 
             final String contact = rc.queryParam("contact").get(0);
             final String newName = rc.queryParam("newName").get(0);
+            if (newName.isEmpty()){
```

Comment:
should we check for `contact` being empty?

Reply:
I'm in two minds about this. I could add a check to every endpoint but now there is no way for a contact to end up equal to "". I also prefer this method because if somehow a `contact` does end up as "" again since this way we can actually change it. Functionally it just throws an error when it can't find the contact.